documentation/modules/rn-2.6.adoc

```diff
@@ -58,6 +58,22 @@ The {project-short} web console can no longer download logs. With this update, y
 
 This part provides features and enhancements introduced in {project-full} 2.6:
 
+.Preservation of static IPs in migrations from vSphere
+
+You can now preserve static IPs of VMs that are installed with a Windows operating system and are migrated from vSphere. link:https://issues.redhat.com/browse/MTV-1148[(MTV-1148)]
+
+.Support for migrating LUKS-encrypted devices in migrations from vSphere
+
+You can now migrate devices that have been encrypted by Linux Unified Key Setup (LUKS) when you create a vSphere migration plan. link:https://issues.redhat.com/browse/MTV-831[(MTV-831)]
+
+.Specifying the primary disk when migrating from vSphere
+
+You can now specify the primary disk when you migrate VMs from vSphere which have more than one single bootable disk. This avoids {project-short} automatically attempting to convert the first bootable disk that it detects while it examines all the disks in a migration plan. This feature is needed because the first bootable disk is not necessarily the disk that the VM is expected to boot from in {virt}. link:https://issues.redhat.com/browse/MTV-1079[(MTV-1079)]
+
+.Providing a link to a remote provider UI
+
+You can now add a link to the UI of a remote cluster when you create a source provider. For example, if the provider is a remote {rhv-full} {rhv-short} cluster, you can add a link to the remote {rhv-short} web console when you define the provider. link:https://issues.redhat.com/browse/MTV-1054[(MTV-1054)]
```

Comment:
@sgratch @yaacov  please check this part (is it something the user adds? I was under the impression that the link is auto-generated from the URL of the provider)

Reply:
I think it's more correct to say:
"You now have a link to the  remote cluster management web based user interface"

reasons:
a. you do not add a link, it's auto generated.
b. you do not add it when you create the provider, you can edit it, in cases the auto generated list is not correct.

p.s.
we can add a few works on the reason for this link, a link to the provider management system makes it easy for users to manage and debug the migration on the source provider side with a direct link from MTV.

Reply:
@ahadas @yaacov @sgratch -- how about this? 


Links to remote provider UIs

You can now remotely access the UI of a remote cluster when you create a source provider. For example, if the provider is a remote {rhv-full} {rhv-short} cluster, {project-short} adds a link to the remote {rhv-short} web console when you define the provider. This feature makes it easier for you to manage and debug a migration from remote clusters.

Reply:
sounds good to me